src/Kernel32/Kernel32+PRODUCT_SUITE.cs

```diff
@@ -90,7 +90,12 @@ public enum PRODUCT_SUITE : short
             /// <summary>
             /// Windows Home Server is installed.
             /// </summary>
-            VER_SUITE_WH_SERVER = unchecked((short)0x00008000)
+            VER_SUITE_WH_SERVER = unchecked((short)0x00008000),
```

Comment:
```
#define VER_SERVER_NT                       0x80000000
#define VER_WORKSTATION_NT                  0x40000000
```

Those values are part of the same "set" of *flags* . Not sure if they are defined elsewhere already. If not, they could probably be added here for completeness.